src/components/Node.tsx

```diff
@@ -55,15 +52,12 @@ export class UnconnectedNode extends React.Component<IExtraProps & INode & INode
             return {
                 nodeID: dbid,
 
-                parent: -1,
+                parent: 1,
```

Comment:
So if we choose to adopt the middle ground I described above, this can be changed back to a nonsense value